net/netifyd/Makefile

```diff
@@ -0,0 +1,83 @@
+# Netify Agent
+# Copyright (C) 2016-2018 eGloo, Incorporated
+#
+# This is free software, licensed under the GNU General Public License v3.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=netifyd
+PKG_VERSION:=2.67
```

Comment:
I see no 2.67 tag on the gitlab repo page.  If you are using explicit tag for version, please leave this out.

I'd like to note that, we generally prefer released, tagged version than specific git commit.

Reply:
Any reason not to use tarballs?
https://gitlab.com/netify.ai/public/netify-agent/tags/v2.86 --> https://gitlab.com/netify.ai/public/netify-agent/-/archive/v2.86/netify-agent-v2.86.tar.bz2

Reply:
@yousong This PR was created early in development and we did not officially release this version.  The updates I'm working on now to meet the requested changes here will be tagged and all future updates will also clone from a tagged commit.  I have removed PKG_VERSION and added the tag (version) to PKG_SOURCE_VERSION.

@diizzyy I want to use tar archives but unfortunately GitLab (and I think most other Git-based Cloud SCMs) do not include submodules in the archives of which we have a handful.  Compilation fails due to missing dependencies.